demisto_sdk/commands/validate/validators/GR_validators/GR107_is_deprecated_content_item_in_usage.py

```diff
@@ -99,18 +99,27 @@ def obtain_invalid_content_items_using_graph(
                 for content_item in content_items
             ]
         )
+        # Group results by content item
+        grouped_results: Dict = {}
+        for item in self.graph.find_items_using_deprecated_items(content_item_paths):
+            for item_using_deprecated in item.content_items_using_deprecated:
+                path_key = str(item_using_deprecated.path.relative_to(CONTENT_PATH))
+                if path_key not in grouped_results:
+                    grouped_results[path_key] = {
+                        "content_object": item_using_deprecated,
+                        "deprecated_items": set(),
+                    }
+                grouped_results[path_key]["deprecated_items"].add(
+                    item.deprecated_item_id
+                )
         return [
             ValidationResult(
                 validator=self,
                 message=self.error_message.format(
-                    deprecated_item_type=item.deprecated_item_type,
-                    deprecated_item=item.deprecated_item_id,
-                    using_deprecated_item=str(
-                        item_using_deprecated.path.relative_to(CONTENT_PATH)
-                    ),
+                    using_deprecated_item=path,
```

Comment:
Not sure the path is necessary here as it should be a part of the error message.